src/main/java/file/TasksLoader.java

```diff
@@ -0,0 +1,84 @@
+package file;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import Tasks.Todo;
+import Tasks.Deadline;
+import Tasks.Event;
+import Tasks.Task;
+
+/**
+ * This class is responsible for loading tasks from a file.
+ * It reads task data from a file, parses the data, and converts it into Task objects.
+ */
+public class TasksLoader {
+    /**
+     * Loads tasks from a file and converts them into Task objects.
+     * It parses each line of the file, identifies the task type, and creates the corresponding Task object.
+     *
+     * @param filePath the file path from which to load the tasks
+     * @return an ArrayList of Task objects loaded from the file
+     * @throws FileNotFoundException if the file cannot be found
+     */
+    public static ArrayList<Task> loadTasks(String filePath) throws FileNotFoundException {
```

Comment:
This function is >30 lines long, this may hinder readability, so you might want to split this function into smaller ones.